src/shared/locales/de/about.js

```diff
@@ -0,0 +1,6 @@
+export default {
+  'engine-version': 'Engine Version',
+  'about': 'Über',
+  'release': 'Veröffentlichte Versionen',
```

Comment:
This "release" is a bit too long, causing the layout to wrap. Is there a way to be shorter?